file.go

```diff
@@ -98,7 +98,7 @@ func (fi *File) Open(flags Flags) (_ FileDescriptor, _retErr error) {
 		// Ok as well.
 	}
 
-	dmod, err := mod.NewDagModifier(context.TODO(), node, fi.dagService, chunker.DefaultSplitter)
+	dmod, err := mod.NewDagModifier(context.TODO(), node, fi.dagService, chunker.SizeSplitterGen(chunker.DefaultBlockSize))
```

Comment:
DefaultSplitter looks the same as SizeSplitterGen(chunker.DefaultBlockSize), could you double check?

Reply:
it is the sme